src/en/contributing.md

```diff
@@ -1,26 +1,34 @@
+Title: Contributing to Juju docs
+
 # Contributing to Juju documentation
 
-Contributing to the Juju documentation is made easy by the format it's
-written in:
-[GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown)
-(or GFM). You will therefore find the source documents straightforward and
-human-readable.
+Juju documentation is published on [jujucharms.com](http://jujucharms.com/docs)
+and hosted on [GitHub](http://github.com). Its source documents are easy to
+understand and edit since they're written, essentially, in standard
+[GitHub Flavored Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/)
+format. Here are a few [cheat](http://askubuntu.com/editing-help)
+[sheets](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for
+writing in GFM.
```

Comment:
In addition there are a few additional conventions we have added to facilitate extra features of the docs - the use of these is explained below

(could be confusing otherwise  - foldouts and admonishments aren't GFM)